src/types.d.ts

```diff
@@ -0,0 +1,10 @@
+import 'racer';
+
+declare module 'racer' {
+  interface util {
+    isProduction: boolean;
+  }
+  interface Model {
+    hasErrored?: boolean;
+  }
+}
```

Comment:
this redundant with typesdefs in other files? (which work?)

Reply:
Yeah, I think this file can be removed now.